devApps/AdalCoreCLRTestApp/Program.cs

```diff
@@ -55,7 +55,7 @@ static void Main(string[] args)
         {
             AuthenticationContext context = new AuthenticationContext("https://login.microsoftonline.com/common", true);
             var certificate = GetCertificateByThumbprint("<CERT_THUMBPRINT>");
-            var result = await context.AcquireTokenAsync("https://graph.windows.net", new ClientAssertionCertificate("<CLIENT_ID>", certificate));
+            var result = await context.AcquireTokenAsync("https://graph.windows.net", new ClientAssertionCertificate("<CLIENT_ID>", certificate), true);
```

Comment:
>true [](start = 146, length = 4)

Thinking the app should include a scenario where this is false (e.g. make a request afterwards where this is false to ensure the cert is not in the request)?

Reply:
good point